README.md

```diff
@@ -83,12 +84,13 @@ c, err := connection.New("127.0.0.1:443", testSpec, readMessageLength, writeMess
 ## Usage
 
 ```go
-// see configuration options for more details about different handlers
+// see configuration options for more details
 c := connection.New("127.0.0.1:9999", brandSpec, readMessageLength, writeMessageLength,
 	connection.SendTimeout(100*time.Millisecond),
 	connection.IdleTime(50*time.Millisecond),
 	connection.PingHandler(pingHandler),
 	connection.UnmatchedMessageHandler(unmatchedMessageHandler),
+	connection.ConnectionClosedHandler(connectionClosedHandler),
```

Comment:
I've seen other projects pass along these in a struct. It allows us to add more without breaking the public API.

Reply:
I saw this can also be provided in the `Options` struct

Reply:
With this one, we pass Option, and we add `ConnectionClosedHandler`, not change it. I believe we don't break public API here.

Reply:
Oh right, I forgot it's `options ...Option`..